pkg/languagedetection/internal/detectors/util.go

```diff
@@ -0,0 +1,13 @@
+package detectors
```

Comment:
missing copyrights

Reply:
Thank you, I actually ended up deleting this file